deploy/evs-csi-plugin/kubernetes/csi-evs-node.yaml

```diff
@@ -0,0 +1,105 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-evs-plugin
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: csi-evs-plugin
+  template:
+    metadata:
+      labels:
+        app: csi-evs-plugin
+    spec:
+      tolerations:
+        - operator: Exists
+      serviceAccount: csi-evs-node-sa
+      containers:
+        - name: evs-driver-registrar
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.4.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/evs.csi.huaweicloud.com/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+            - name: registration-dir
+              mountPath: /registration
+        - name: liveness-probe
+          image: k8s.gcr.io/sig-storage/livenessprobe:v2.5.0
+          args:
+            - --csi-address=/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: evs-csi-plugin
+          securityContext:
+            privileged: true
+          image: swr.cn-north-4.myhuaweicloud.com/k8s-csi/evs-csi-plugin:v0.0.1

```

Comment:
after confirmation, the first version is `v0.1.0`

Reply:
done